### ISSUE TITLE
Training detail

### ISSUE THREAD

**Issue by zhlhlhlhl** [PKU-YuanGroup/Chat-UniVi#65]
Hi, really impressive work! I'm curious why using zero3 in pretraining but zero2 in instruction training?

**Comment by jpthu17** [comment]
We find using zero3 is very easy to hang because the lengths of the video and image data vary greatly.

**Comment by zhlhlhlhl** [comment]
Yes, it's true. Besides, in the instruction training phase, I met  CUDA out of memory error in 5258/8056. Are your experiments conducted on A100 GPUs with 40GB or 80GB
